src/components/Stack/tests/Stack.test.tsx

```diff
@@ -0,0 +1,254 @@
+import * as React from 'react';
+import {mount} from 'enzyme';
+import Stack from '../Stack';
+import Badge from '../';
+import Item from '../Item';
+import Heading from '../';
+
+describe('Stack Component Suit', () => {
+    describe('Test children property', () => {
+        it('should verify children property is defined', () => {
+            const stackWrapper = mount(
+                                        <Stack>
+                                          <Badge>Paid</Badge>
+                                          <Badge>Fulfilled</Badge>
+                                        </Stack>,
+                                 );
+            expect(stackWrapper.childAt(0).prop('children').props.children).toBe('Paid');
```

Comment:
just a tiny thing. I would probably push line 12 up to line 11 after the parenthesis. Something like 

+            const stackWrapper = mount(<Stack>